draft-ietf-lpwan-schc-over-lorawan.md

```diff
@@ -392,12 +394,8 @@ The mechanism for sharing those RuleID values is outside the scope of this docum
 
 ## IID computation
 
-As LoRaWAN network uses unique EUI-64 per end-device, the Interface IDentifier is
-the LoRaWAN DevEUI.
-It is compliant with [RFC4291] and IID starting with binary 000 must enforce
-the 64-bit rule.
-
-TODO: Derive IID from DevEUI with privacy constraints ? Ask working group ?
+It is RECOMMENDED to create Interface IDentifier following
+{{I-D.ietf-lpwan-ipv6-static-context-hc}}, [rfc7217] and [rfc8064]
 
```

Comment:
I am not sure it is necessary to refer to RFC8064 which only purpose seems to point to RFC7217.

Otherwise, I agree with the change. 👍

Reply:
I agree, but we should add some text for the security considerations IMO. I can try to take a stab at it in the next 1-2 days.

Reply:
It seems that at least we should mention [RFC8065][1].

[1]: https://datatracker.ietf.org/doc/rfc8065/

Reply:
@jca-klk : Why removing RFC8064, it explains pro and cons of all method

@ivajloip : ok, thank you for the proposition :)

Reply:
My advice would be to ask the IETF gurus for advice 😃 RFC8065 and RFC8064 are really short and worth reading anyway.

Reply:
So we met Dave Thaler with Dominique, he recommended us to specify in the profile how to generate IID, and how to do it. I will do a proposition this week

Reply:
ok, thanks for letting me know :slightly_smiling_face: